pkg/volume/rbd/rbd_util.go

```diff
@@ -268,8 +281,9 @@ func (util *RBDUtil) AttachDisk(b rbdMounter) error {
 
 	devicePath, found := waitForPath(b.Pool, b.Image, 1)
 	if !found {
-		// modprobe
-		_, err = b.plugin.execCommand("modprobe", []string{"rbd"})
+		// modprobe on the host
+		exe := exec.New()
```

Comment:
That's a question... Where is the kernel module? On the host or in a container? It's hard to ship modules in containers because you don't know what kernel runs on the host. Shall we require modules in the container (e.g. by bind-mounting `/lib/modules` from the host to the container)?